README.md

```diff
@@ -286,6 +159,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] The MIT link in your Readme file leads to a 404 Error page, I'll suggest you add a license file to your project and link it properly as broken links in the readme make your repository look a bit unprofessional. :smile: :small_airplane: